auth/providers/azure/options.go

```diff
@@ -74,7 +74,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.AKSTokenURL, "azure.aks-token-url", "", "url to call for AKS OBO flow")
 	fs.StringVar(&o.POPTokenHostname, "azure.pop-hostname", "", "hostname used to run the pop hostname verification; 'u' claim")
 	fs.BoolVar(&o.EnablePOP, "azure.enable-pop", false, "Enabling pop token verification")
-	fs.DurationVar(&o.PoPTokenValidityDuration, "azure.pop-token-validity-duration", 15, "time duration for PoP token to be considered valid from creation time, default 15 min")
+	fs.DurationVar(&o.PoPTokenValidityDuration, "azure.pop-token-validity-duration", 15*time.Minute, "time duration for PoP token to be considered valid from creation time, default 15 min")
```

Comment:
Passing in nanoseconds (`time.Duration`'s underlying unit) seems unexpected for a CLI argument. Maybe we can change the type here to be `int` and rename the variable to `PoPTokenValidityMinutes`? Then do the conversion to `time.Duration` later.

Reply:
`Passing in nanoseconds (time.Duration's underlying unit) seems unexpected for a CLI argument` why is that ? there is a `fs.DurationVar` we should just use it. This also allows to pass seconds / minutes / hours if needed.

Reply:
nvm, it seems that Go is able to parse `15m` into a duration so users can pass it via a human-readable format.